### ISSUE TITLE
[Bug]: if folder_name is given, the microos snapshot build fails because of cd 

### ISSUE THREAD

**Issue by schlichtanders** [kube-hetzner/terraform-hcloud-kube-hetzner#746]
### Description

These lines in create.sh will append the folder_name to the folder_path
https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/blob/209d48a1a56e8e8e380e8aec67a64f3d588c2621/scripts/create.sh#L22-L25

however when creating the microos snapshot, the `cd` again appends the folder_name to the folder_path, which apparently is one append too much.
https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/blob/209d48a1a56e8e8e380e8aec67a64f3d588c2621/scripts/create.sh#L51

the needed changes would be
```diff
- cd "${folder_path}/${folder_name}" && packer build hcloud-microos-snapshots.pkr.hcl
+ cd "${folder_path}" && packer build hcloud-microos-snapshots.pkr.hcl
```

### Kube.tf file

```terraform
have not reached this stage yet
```

### Platform

Linux

**Comment by mysticaltech** [comment]
Good catch @schlichtanders, next time don't hesitate to do a quick PR, it's always very much appreciated! 🙏

**Comment by mysticaltech** [comment]
Fixed in v2.1.2 🚀
